index.html

```diff
@@ -347,5 +347,104 @@ <h2>Examples</h2>
       </pre>
     </div>
   </section>
+  <section>
+    <h2>Transferable MediaStreamTrack</h2>
+    <div>
+      <p>{{MediaStreamTrack}} are <a data-cite="!HTML/#transferable-objects">transferable objects</a>.
+      This allows manipulating real-time media outside the context it was requested or created, for instance in workers or third-party iframes.</p>
+      <p>To preserve the existing privacy and security infrastructure, in particular for capture tracks,
+      a transferred track remains tightly bound to the original track.
+      In particular, if the original track gets muted, unmuted or ended, the transferred track will automatically be muted, unmuted or ended.
+      It is in particular important to ensure that media will stop flowing as soon as the context of the original track goes away.</p>
+    </div>
+    <div>
+      <p>The WebIDL changes are the following:
+      <pre class="idl"
+>[Exposed=(Window,Worker,AudioWorklet), Transferable]
+partial interface MediaStreamTrack {
+// Remove partial once we figure out how to do it without creating respec warnings.
+// No change to MediaStreamTrack exposed API.
```

Comment:
How can we remove `partial`? This isn't the full interface.

This might be one of those cases where we need to say "wait for the document to be merged with mediacapture-main"

Reply:
Agreed, that is the best tradeoff I found to still add useful IDL information in that PR.

Reply:
I don't see why partial should be removed, given that this is only part of the interface, and that the WebIDL convention is that all IDL that a browser implements should be considered merged in the implementation.
Having an extended visibility list in a partial is unusual for sure, but if the IDL parser doesn't complain, I'm not going to.